indian-languages/README.md

```diff
@@ -4,6 +4,6 @@ Most Indian languages use Bharati Braille a system developed in the 1950's to un
 
 The documents here were supplied by Dipendra Manocha of The Braille Council of India. They are images so not accessible.
 
-`Braille chart Pg-1.jpg` and `Braille chart pg-2.jpg` show a summary of the Bharati Braille system. For how it applies to individual languages refer to [World Braille Usage (3rd edition)](http://liblouis.org/braille-specs/world-braille-usage-3rd-edition.pdf).
+`Braille chart Pg-1.jpg` and `Braille chart pg-2.jpg` show a summary of the Bharati Braille system. For how it applies to individual languages refer to [World Braille Usage (3rd edition)](https://github.com/liblouis/braille-specs/blob/master/world-braille-usage-third-edition.pdf).
```

Comment:
Hmm this is not a good idea I think. Why not just have "http://liblouis.org/braille-specs/#world-braille-usage-3rd-edition" (like you had a few revisions ago)?

Reply:
Yes you are right - done (I think I was thinking it would be good to link to actual pdf)